skimage/filters/rank/core_cy.pxd

```diff
@@ -1,4 +1,4 @@
-from numpy cimport uint8_t, uint16_t, float32_t, double_t
+from numpy cimport uint8_t, uint16_t, float32_t, float64_t
```

Comment:
There are a few more instances of `double_t` throughout the code base, e.g. in `_ncut_cy.pyx` or `_hough_transform.pyx`. @grlee77 do you want to catch these as well?

Reply:
Yeah, I think I must have mistyped something when checking for doubles yesterday. I am updating it now.